tools/MatrixHelper/InstrumentationData.cs

```diff
@@ -24,48 +24,50 @@ public static Instrumentation[] GetInstrumentations()
-            new("process.runtime.dotnet.jit.methods_compiled.count", counter, "Number of times the JIT compiler compiled a method since the process started. Only available for .NET 8."),
-            new("process.runtime.dotnet.jit.compilation_time", counter, "Amount of time the compiler spent compiling methods since the process started. Only available for .NET 8."),
-            new("process.runtime.dotnet.timer.count", upDownCounter, "Number of timer instances currently active. Only available for .NET 8."),
-            new("process.runtime.dotnet.assemblies.count", upDownCounter, "Number of .NET assemblies that are currently loaded. Only available for .NET 8. and .NET Framework."),
-            new("process.runtime.dotnet.exceptions.count", counter, "Count of exceptions thrown in managed code since the observation started. Only available for .NET 8. and .NET Framework."),
+            new("dotnet.process.cpu.time", counter, "CPU time used by the process. Only available for .NET 9.", conditionallyBundled),
+            new("dotnet.process.memory.working_set", upDownCounter, "The number of bytes of physical memory mapped to the process context. Only available for .NET 9.", conditionallyBundled),
+            new("dotnet.gc.collections", counter, "The number of garbage collections that have occurred since the process has started. Only available for .NET 9.", conditionallyBundled),
+            new("dotnet.gc.heap.total_allocated", counter, "The approximate number of bytes allocated on the managed GC heap since the process started. The returned value does not include any native allocations. Only available for .NET 9.", conditionallyBundled),
+            new("dotnet.gc.last_collection.memory.committed_size", upDownCounter, "The amount of committed virtual memory in use by the .NET GC, as observed during the latest garbage collection. Only available for .NET 9.", conditionallyBundled),
+            new("dotnet.gc.last_collection.heap.size", upDownCounter, "The managed GC heap size (including fragmentation, conditionallyBundled, true), as observed during the latest garbage collection. Only available for .NET 9.", conditionallyBundled),
+            new("dotnet.gc.last_collection.heap.fragmentation.size", upDownCounter, "The heap fragmentation, as observed during the latest garbage collection. Only available for .NET 9.", conditionallyBundled),
+            new("dotnet.gc.pause.time", counter, "The total amount of time paused in GC since the process started. Only available for .NET 9.", conditionallyBundled),
+            new("dotnet.jit.compiled_il.size", counter, "Count of bytes of intermediate language that have been compiled since the process started. Only available for .NET 9.", conditionallyBundled),
+            new("dotnet.jit.compiled_methods", counter, "The number of times the JIT compiler compiled methods since the process started. Only available for .NET 9.", conditionallyBundled),
```

Comment:
```suggestion
            new("dotnet.jit.compiled_methods", counter, "The number of times the JIT compiler (re)compiled methods since the process started. Only available for .NET 9.", conditionallyBundled),
```